benefits/eligibility/views.py

```diff
@@ -53,7 +53,7 @@ def index(request):
     ctx = page.context_dict()
     ctx["help_page"] = help_page
     ctx["help_text"] = format_html(
-        _("eligibility.pages.index.help_text%(help_link)s") % {"help_link": f"{help_page}#what-is-cal-itp"}
+        _("eligibility.pages.index.help_text%(help_link)s") % {"help_link": f"{help_page}#login-gov"}
```

Comment:
I'm not sure if this change is correct (but I also agree that the existing anchor link probably shouldn't be there).

This line is for this link:

![image](https://user-images.githubusercontent.com/1783439/199619266-9e0cd0e5-5e4b-40d2-ac55-f3532f73f99f.png)

Shouldn't that just be to the generic Help Page, no anchor?

Reply:
The top of the page already links to the generic Help Page (What is Cal-ITP?) question. So I linked the What Is Login.gov? (followed by What is Courtesy Card?) section here so people can be taken to the part about the two options. We don't actually have anything on the Help page that directly answers which option is right for people though.

Reply:
Hmm... I see your point, but it still feels confusing to me to direct the user to Login.gov when they haven't made a selection yet. And SEO/accessibility-wise, this link mentioning Login.gov is not exactly accurate...

Reply:
Yeah. I guess for now I'll just remove the anchor entirely cos there aren't any other good questions to link to.